vllm/_custom_ops.py

```diff
@@ -153,6 +153,36 @@ def marlin_gemm(a: torch.Tensor, b_q_weight: torch.Tensor,
                                 size_n, size_k)
 
 
+# cutlass
+def cutlass_scaled_mm_dq(
+        a: torch.Tensor,
+        b: torch.Tensor,
+        a_scales: torch.Tensor,
+        b_scales: torch.Tensor,
+        out_dtype: Type[torch.dtype] = torch.bfloat16) -> torch.Tensor:
+    shape_fallback = b.shape[0] % 16 != 0 or b.shape[1] % 16 != 0
```

Comment:
I think it would be better to remove the fallback and just replace this with an assert (similar for `capability < 75`). The reason is that it can make performance unpredictable. For this low level method, the caller asks for `cutlass` (that's in the name), so they should get cutlass. Falling back should be done on a higher level if needed :)

Reply:
Agreed, much better this way